.gitignore

```diff
@@ -13,3 +13,8 @@ profile
 *.moved-aside
 .DS_Store
 *.xccheckout
+# CocoaPods
+Podfile.lock
+Pods/
```

Comment:
Regarding the `Pods` folder there are two ways to go (commit it or not) and both solutions have pros and cons, but I personally prefer to commit them (I agree that that's a personal and debatable choice)

_Note that whatever your choice is regarding the `Pods/` folder, that does not change anything about the fact that you still must always commit `Podfile.lock` whatever policy you choose._